gcloud_bigtable/google/bigtable/v1/bigtable_service_api.py

```diff
@@ -0,0 +1,175 @@
+# Copyright 2015 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# EDITING INSTRUCTIONS
+# This file was generated from google/bigtable/v1/bigtable_service.proto,
+# and updates to that file get reflected here through a regular refresh
+# process. However, manual and updates to that file get reflected here
+# through a regular refresh additions are allowed because the refresh
+# process performs a 3-way merge in order to preserve those manual additions.
+# In order not to break the refresh process, only certain types of
+# modifications are allowed.
+#
+# Allowed modifications:
+# 1. New methods (these should be added to the end of the class)
+# 2. "Notes specific to this wrapper method" sections in the method
+#    documentation
+#
+# Happy editing!
+
+from google.bigtable.v1 import bigtable_data_pb2
+from google.bigtable.v1 import bigtable_service_messages_pb2
+from google.bigtable.v1 import bigtable_service_pb2
+from google.gax import api_callable
+from google.gax import api_utils
+from google.gax import page_descriptor
+
+class BigtableServiceApi(object):
+  """Service for reading from and writing to existing Bigtables."""
```

Comment:
@anthmgoogle It seems referring to them as "Bigtables" instead of "Tables" is how things are done by the people writing the `.proto` defs, so maybe that is the "standard term".

It's unclear from the external API docs.